specification/cognitiveservices/Speech.VideoTranslation/main.tsp

```diff
@@ -321,3 +417,26 @@ interface OperationOperations {
   @doc("Fetch a operation by ID.")
   getOperation is ops.ResourceRead<Operation>;
 }
+
+@added(ApiVersions.v2025_05_20)
+interface EventHubConfigOperations {
+  @doc("Create or replace EventHub configuration.")
+  @route("/configurations/event-hub")
+  @put
+  createOrReplaceEventHubConfig is RpcOperation<EventHubConfig, EventHubConfig>;
+
+  @doc("Get EventHub configuration.")
+  @route("/configurations/event-hub")
+  @get
+  getEventHubConfig is RpcOperation<{}, EventHubConfig>;
+
+  @doc("Get EventHub configuration.")
+  @delete
+  @route("/configurations/event-hub")
+  deleteEventHubConfig is RpcOperation<{}, NoContentResponse>;
+
+  @doc("Send ping to EventHub to verify configuration.")
+  @route("/configurations/event-hub:ping")
+  @post
+  pingEventHub is RpcOperation<{}, OkResponse>;
```

Comment:
@markcowl Can we have your opinion here? The `{}` here to signify "no request body" works for Swagger (`void` does not pass lint), but may not be good TypeSpec.